src/runtime-rs/crates/hypervisor/src/ch/inner_hypervisor.rs

```diff
@@ -908,6 +906,59 @@ fn get_guest_protection() -> Result<GuestProtection> {
     Ok(guest_protection)
 }
 
+// Return a TID/VCPU map from a specified /proc/{pid} path.
+fn get_ch_vcpu_tids(proc_path: &str) -> Result<HashMap<u32, u32>> {
+    let mut vcpus = HashMap::new();
```

Comment:
Nit: Can you move this line to just above the `for` loop.

Reply:
ah good catch, thanks @jodh-intel